packages/patternfly-4/react-core/src/experimental/components/OverflowMenu/OverflowMenu.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/OverflowMenu/overflow-menu';
+import { css, getModifier } from '@patternfly/react-styles';
+import { OverflowMenuContext } from './OverflowMenuConstants';
+
+export interface OverflowMenuProps extends React.HTMLProps<HTMLDivElement> {
+  /** Any elements that can be rendered in the menu */
+  children?: any;
+  /** Additional classes added to the OverflowMenu. */
+  className?: string;
+  /** Indicates breakpoint at which to switch between horizontal menu and vertical dropdown */
+  breakpoint: 'md' | 'lg' | 'xl';
+}
+
+export const OverflowMenu: React.SFC<OverflowMenuProps> = ({
+  className,
+  breakpoint,
+  children,
+  ...props
+}) => {
+  const breakpointSize = `${breakpoint[0].toUpperCase()}${breakpoint.substr(1)}`;
```

Comment:
this is fine, but looking at the getModifier function it checks for the modifier also by using the camelCase library, which I believe would turn the string `showOn md` into `showOnMd` with a simple space between the words :)

Reply:
Good to know!